### ISSUE TITLE
Enforce block rules

### ISSUE THREAD

**Issue by greatislander** [accessibility-exchange/platform#585]

**Comment by jobara** [comment]
@greatislander, just to clarify, this is different than the suspending accounts that an admin can currently do?

**Comment by greatislander** [comment]
Yes— it's users blocking other users/orgs. There might be a valid reason why an individual never wants to interact with, say, a bank or other regulated organization that they've had bad experiences with that doesn't mean that regulated org needs to be suspended. Same with individuals and other individuals.
